model/src/main/java/org/cbioportal/model/MolecularProfile.java

```diff
@@ -84,7 +84,9 @@ public MolecularAlterationType getMolecularAlterationType() {
         // https://github.com/cBioPortal/cbioportal/blob/8704058562c386afeac3082e50f39c1097d47983/core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java#L93
         // But somehow there was no migration for existing data. To resolve it replace FUSION alteration type by STRUCTURAL_VARIANT.
```

Comment:
Maybe we can also add some comment at here to mention that "fusion data could also come from a mutation profile"